WalletWasabi.Gui/ViewModels/StatusBarViewModel.cs

```diff
@@ -308,11 +304,19 @@ private void OnResponseArrivedIsGenSocksServFail(bool isGenSocksServFail)
 						MainWindowViewModel.Instance.ShowDialogAsync(new GenSocksServFailDialogViewModel()).GetAwaiter().GetResult();
 					}
 				}
+				else
```

Comment:
Makes no sense. Delete it.

Reply:
I agree. I simply thought they were writing to emphasize to not do anything.